lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -46,6 +46,30 @@ def cockroachdb_connection(config)
 
 module ActiveRecord
   module ConnectionAdapters
+    module CockroachDBConnectionPool
+      def initialize(pool_config)
+        super(pool_config)
+        disable_telemetry = pool_config.config[:disable_cockroachdb_telemetry]
+        adapter = pool_config.config[:adapter]
+        return if disable_telemetry || adapter != "cockroachdb"
+
+        with_connection do |conn|
+          if conn.active?
+            begin
+              query = "SELECT crdb_internal.increment_feature_counter('ActiveRecord %d.%d')"
+              conn.execute(query % [ActiveRecord::VERSION::MAJOR, ActiveRecord::VERSION::MINOR])
+            rescue ActiveRecord::StatementInvalid
+              # The increment_feature_counter built-in is not supported on this
+              # CockroachDB version. Ignore.
+            rescue StandardError => e
+              conn.logger.warn "Unexpected error when incrementing feature counter: #{e}"
+            end
+          end
```

Comment:
We wrapped this in a `begin/rescue` block for version 6.x. We should probably do that here as well.

Reply:
isn't that there already? line 64 rescues all StandardErrors, which seems to be what 6.x is doing

Reply:
We found out that `with_connection` would raise errors if the target database didn't exist and had to wrap that block with `rescue` as well (#206).

Reply:
ah, i see. i missed that PR since i hadn't fetched the latest changes